packages/components/field-label/src/field-label.tsx

```diff
@@ -14,6 +14,7 @@ import Constraints from '@commercetools-uikit/constraints';
 import Stack from '@commercetools-uikit/spacings-stack';
 import Inline from '@commercetools-uikit/spacings-inline';
 import Label from '@commercetools-uikit/label';
+import { useTheme } from '@commercetools-uikit/design-system';
```

Comment:
Please remember to add the `// TODO: @redesign cleanup` comment at the top of the files where we're using `useTheme` hook.